cmd/kubeapps-apis/plugins/pkg/clientgetter/clientgetter.go

```diff
@@ -23,6 +23,17 @@ type HelmActionConfigGetterFunc func(ctx context.Context, namespace string) (*ac
 type ClientGetterFunc func(ctx context.Context, cluster string) (kubernetes.Interface, dynamic.Interface, error)
 type ClientGetterWithApiExtFunc func(context.Context) (kubernetes.Interface, dynamic.Interface, apiext.Interface, error)
 
+// TODO: (gfichtenholt) we might want to consider some kind of
+// type Clients struct {
+//   kubernetes.Interface
+//   dynamic.Interface
+//   apiext.Interface
+// }
```

Comment:
Yep, good idea. As we continue to add more and more clients, it's becoming quite misleading just to rely on the number of returned items (especially because of the differences between every plugin).